Examples/Java/Sources/Board.java

```diff
@@ -312,4 +318,71 @@ public void mergeFrom(Board model) {
         }
     
     }
+    public static class BoardTypeAdapterFactory implements TypeAdapterFactory {
+    
+    
```

Comment:
Extra newlines? Maybe just happens when there aren’t instance variables?

Reply:
Fixed. Did a general clean up of tabbing and spaces.